static/modules/gamemanager.js

```diff
@@ -141,8 +147,16 @@ class GameManager {
   }
 
   togglePause() {
-    toggleGuiElement("pausescreen");
-    this.gamePause = !this.gamePause;
+    if (!this.gamePause) {
+      gameManager.guiElements.get("pausescreen").active = true;
+      this.gamePause = true;
+    } else {
```

Comment:
This is gross...  can menus have a parent-child relation where closing the parent closes the child?

Reply:
I changed some of this logic in ccc3e9f